### ISSUE TITLE
Consider pulling in some options from Chromium's Accessibility settings page

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#15844]
## Description 

Consider pulling in some options from Chromium's Accessibility settings page; filed this issue after speaking briefly to @rebron.

## Steps to Reproduce

   1. load `chrome://settings/accessibility` in Chrome/Chromium
   2. note the available Accessibility options
   3. load `brave://settings/` in Brave
   4. note the lack of available Accessibility options

## Actual result:

We are missing the following:
* `Live Caption`
* `Caption preferences`
* `Show a quick highlight on the focused object`
* `Navigate pages with a text cursor`
* `Add accessibility features`

<img width="1505" alt="Screen Shot 2021-05-13 at 1 18 37 PM" src="https://user-images.githubusercontent.com/387249/118182358-c0e84280-b3ed-11eb-8c40-1fc2e45e17e0.png">

`Live Caption` -- as well as a few others -- are available, instead, as feature flags:

<img width="1505" alt="Screen Shot 2021-05-13 at 1 16 51 PM" src="https://user-images.githubusercontent.com/387249/118182604-0f95dc80-b3ee-11eb-802a-2f80e4e7a2f8.png">

## Expected result:
<img width="1312" alt="Screen Shot 2021-05-13 at 1 10 13 PM" src="https://user-images.githubusercontent.com/387249/118182161-867ea580-b3ed-11eb-9cc5-7c5050bcd3c6.png">

Do we want to pull in, at least:
* `Show a quick highlight on the focused object`
* `Navigate pages with a text cursor`

`Live Caption` and associated settings downloads files from Google, so not sure if we want that, and if so, if we want to proxy that?

Thoughts?

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.26.18 Chromium: 91.0.4472.38 (Official Build) nightly (x86_64)
-- | --
Revision | 8155e7afab5d695cf0e028f4d77203287523cda9-refs/branch-heads/4472_35@{#6}
OS | macOS Version 11.3.1 (Build 20E241)

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the nightly channel? yes

**Comment by karenkliu** [comment]
@rebron Designs added!

**Comment by farline99** [comment]
I hate this buttons highlight so much........

**Comment by Rogue-Git-Dev** [comment]
To this day the accessibility section is not existent in Brave's settings. This must change. Brave must incorporate more accessibility in the very least to have feature parity with Chromium.

**Comment by LaurenWags** [comment]
Labelling as `QA/Blocked` as we need screenshots from all OSes (Win, macOS, Linux) to verify against. Per @rebron he will provide these as the different OSes show different options on the accessibility settings page.

**Comment by MadhaviSeelam** [comment]
Verification `Passed` using
```
Brave | 1.78.82 Chromium: 135.0.7049.84 (Official Build) beta (64-bit)
-- | --
Revision | 4b464ec3ff31141fcfa4c0d3e38e0ac0ac14ddbb
OS | Windows 11 Version 24H2 (Build 26100.3476)
```

Verified using the STR from the description https://github.com/brave/brave-browser/issues/15844#issue-891346894

Confirmed `brave://settings/accessibility` matched to expected results screenshot https://github.com/brave/brave-browser/issues/15844#issue-891346894

![Image](https://github.com/user-attachments/assets/88af3319-fa65-4457-91be-dd5301747859)

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.78.86 Chromium: 135.0.7049.100 (Official Build) beta (x86_64)
-- | --
Revision | f5d151b3a6fb9511768159f0856a79a90ee50d1d
OS | macOS Version 14.7.5 (Build 23H527)
```

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave	1.78.88 Chromium: 136.0.7103.34 (Official Build) beta (64-bit) 
Revision	c3708d957bbde3ce51a507be8cdd52b9cb880ba3
OS	Linux
```
Verified the issue using  https://github.com/brave/brave-browser/issues/15844#issue-891346894

Confirmed brave://settings/accessibility page loaded as expected and matched to the expected results
Spot checked some of the settings on the page and they worked as expected.

![Image](https://github.com/user-attachments/assets/cbc60b24-e2d3-4403-a2f8-0a451403d8ab)
